examples/process-manager-plugin/bootstrap/main.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/ligato/cn-infra/agent"
+	"github.com/ligato/cn-infra/logging"
+	bs "github.com/ligato/cn-infra/processes/bootstrap"
+)
+
+func main() {
+	// The bootstrap plugin defines a configuration file allowing to easily manage processes using process
+	// manager plugin.
+	//
+	// The config file can be put to the bootstrap either via flag "-process-bootstrap-config="
+	// or define its path in the environment variable "PROCESS-BOOTSTRAP_CONFIG". Another option is
+	// to define config directly with "UseConf()" - this option will be shown in the example. A sample of
+	// YAML config file can be found in the processes/bootstrap folder.
+
+	log := logging.DefaultLogger
+
+	// Define three processes p1-p3 where the third process is set to terminate p2 if stopped
+	conf := &bs.Config{
+		Processes: []bs.Process{
+			{
+				Name:        "p1",
+				LogFilePath: "example.log",
+				BinaryPath:  "../test-process/test-process",
+				Args:        []string{"-max-uptime=60"},
+			},
+			{
+				Name:        "p2",
+				LogFilePath: "example.log",
+				BinaryPath:  "../test-process/test-process",
+			},
+			{
+				Name:           "p3",
+				LogFilePath:    "example.log",
+				BinaryPath:     "../test-process/test-process",
+				TriggerStopFor: []string{"p2"},
+			},
+		},
+	}
+
+	// start plugin
+	bsp := bs.NewPlugin(bs.UseConf(*conf))
+
+	go func() {
```

Comment:
You could actually call `a.Start()` together with `defer a.Stop()` here and avoid using goroutine and sleep.